routing/http/client/client.go

```diff
@@ -638,3 +638,81 @@ func (c *Client) PutIPNS(ctx context.Context, name ipns.Name, record *ipns.Recor
 
 	return nil
 }
+
+// GetClosestPeers obtains the closest peers to the given peer ID.
+func (c *Client) GetClosestPeers(ctx context.Context, peerID peer.ID) (peers iter.ResultIter[*types.PeerRecord], err error) {
+	m := newMeasurement("GetClosestPeers")
+
+	// Build the base URL path
+	u, err := gourl.JoinPath(c.baseURL, "routing/v1/dht/closest/peers", peer.ToCid(peerID).String())
```

Comment:
Maybe the closest peers URL path should be a const defined at top of file, to make it easier to find and change.

Reply:
Prior art is like this for the other methods... given the constant is used in a single place and that place is the method that it is relevant to, it's probably not adding much.. so I'd rather leave it.